dart/lib/src/protocol/sentry_exception.dart

```diff
@@ -0,0 +1,44 @@
+import 'mechanism.dart';
+import 'sentry_stack_trace.dart';
+
+class SentryException {
```

Comment:
missing `threadId` field

https://github.com/getsentry/sentry-java/blob/5ebd49cbfd17eb66175e9247db6d27e4df84a56b/sentry/src/main/java/io/sentry/protocol/SentryException.java#L12

Reply:
✅ Dart doesn't expose the thread, but I added the field